### ISSUE TITLE
Incorrect Patched Version 

### ISSUE THREAD

**Issue by kbexit** [revanced/revanced-manager#904]
### Type

Other

### Bug description

Revanced manager says my apps are older than they actually are. Go into the individual app settings and app info, it tells me the correct version. Minor issue but an annoyance for sure. 

### Steps to reproduce

Deleted and reinstalled, same issue. 

### Android version

Android 13 Stock Pixel

### Manager version

Version: 1.3.0
Model: Pixel 6 Pro
Android Version: 13
Arch: [arm64-v8a, armeabi-v7a, armeabi]

### Target package name

com. google. android. youtube
com. google. android. apps . youtube.music
Com. twitter . android

### Target package version.

version 18.19.35 (YouTube)
version 5.39.52 (YouTube Music)
version 9.90.0-release.0 (Twitter)

### Installation type

Non-root

### Patches selected.

YouTube:
Always Autorepeat
Client Spoof
Comments
Copy Video Url
Custom Branding
Disable Auto Captions
Disable Fullscreen Panels
Disable Player Popup Panels
Disable Shorts On Startup
Disable Zoom Haptics
Downloads
Enable Debugging
Hdr Auto Brightness
Hide Ads
Hide Album Cards
Hide Artist Card
Hide Autoplay Button
Hide Breaking News Shelf
Hide Captions Button
Hide Cast Button
Hide Crowdfunding BoX
Hide Email Address
Hide Endscreen Cards Hide Filter Bar
Hide Floating Microphone Button
Hide Get Premium
Hide Info Cards
Hide Load More Button
Hide Player Buttons
Hide Player Overlay •Hide Seekbar
Hide Shorts Components Hide Timestamp
Hide Video Action Buttons Hide Watch In Vr
•Hide Watermark
Minimized Playback
Navigation Buttons
Old Quality Layout
Open Links Externally ·Premium Heading
Remember Video Quality Return Youtube Dislike
Seekbar Tapping
Sponsorblock
•Spoof App Version
Swipe Controls
•Tablet Mini Player
Theme
Vanced MicroG Support Video Ads
Video Speed
Wide Searchbar

YouTube Music: 
Says none (?) I'll retry this again 

Twitter: 
Dynamic Color
Hide Ads
Hide Recommended Users

### Device logs (exported using Manager settings).

```shell
--------- beginning of main
05-26 10:01:04.641 32254 32254 E SurfaceSyncer: Failed to find sync for id=0
05-26 10:01:04.641 32254 32254 E SurfaceSyncer: Failed to find sync for id=1
05-26 10:01:04.641 32254 32254 E SurfaceSyncer: Failed to find sync for id=2
05-26 10:01:04.641 32254 32254 E SurfaceSyncer: Failed to find sync for id=3
05-26 10:01:04.641 32254 32254 E SurfaceSyncer: Failed to find sync for id=4
05-26 10:01:04.641 32254 32254 E SurfaceSyncer: Failed to find sync for id=5
05-26 10:01:04.641 32254 32254 E SurfaceSyncer: Failed to find sync for id=6
```

### Screenshots or videos

![Screenshot_20230526-100243](https://github.com/revanced/revanced-manager/assets/102698001/4382cf47-4ece-425b-bf23-2e146d580456)
![Screenshot_20230526-100235](https://github.com/revanced/revanced-manager/assets/102698001/0cadce70-a194-41d9-a1c7-52e6c5eb377f)
![Screenshot_20230526-100207](https://github.com/revanced/revanced-manager/assets/102698001/136c3978-b122-4e65-a0ae-18df1b8f0f4d)
![Screenshot_20230526-100202](https://github.com/revanced/revanced-manager/assets/102698001/d3169659-7adb-4d6c-bc58-852b5a2dd33c)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] The issue is related solely to the ReVanced Manager and not related to patching errors or patches

**Comment by SilverGreen93** [comment]
I can confirm this on my Xiaomi Mi 10T Pro, MIUI 14, Android 12

**Comment by ILoveOpenSourceApplications** [comment]
I believe this is related to #692.

**Comment by kbexit** [comment]
> I believe this is related to #692.

I probably didn't see it because of the title being: 
Bug: 'potentially fix apps disappearing when update is available

**Comment by ILoveOpenSourceApplications** [comment]
I named it after the patch which was released to fix an issue, where no patched applications were showing up. Now the apps are showing up thanks to that patch but for some reason the version number and date of update/install isn't changing whenever the user updates the respective application.

**Comment by kbexit** [comment]
> I named it after the patch which was released to fix an issue, where no patched applications were showing up. Now the apps are showing up thanks to that patch but for some reason the version number and date of update/install isn't changing whenever the user updates the respective application.

Oh you know what! That might be what my issue is? I have stock pixel ROM (can't afford to buy my phone out since I'm a single income dad) and I disabled my YouTube apps system level to favor Revanced. Maybe it's pulling version from the disabled apps?

**Comment by ILoveOpenSourceApplications** [comment]
Can you confirm whether your disabled YouTube app version matches with the ReVanced YouTube version shown inside the manager. In my experience, the app version remains that of the first YouTube version the user patches and installs via the manager. And in your case that's 18.05.40 and not that of the disabled app version.

**Comment by kbexit** [comment]
> Can you confirm whether your disabled YouTube app version matches with the ReVanced YouTube version shown inside the manager. In my experience, the app version remains that of the first YouTube version the user patches and installs via the manager. And in your case that's 18.05.40 and not that of the disabled app version.

Turns out, the disabled app is more updated than what Revanced manager shows too... 
Don't know where these numbers are coming from, but it sure is odd.

Also should have known this isn't the case since twitter doesn't come stock on pixel 😅

**Comment by EvadeMaster** [comment]
🎉 The fix for this issue have been in #1022 and released in ReVanced Manager v1.4.1

If problem persist, feels free to reopen this issue!
